### PR TITLE
Fix direct access error of wp-includes blocks index file

### PR DESCRIPTION
**Pull request by itzmekhokan**
Fix direct accessibility of of `/wp-includes/blocks/index.php`

Trac ticket: [#60352](https://core.trac.wordpress.org/ticket/60352)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/blocks/index.php [hunk 1]
```diff
@@ -5,6 +5,10 @@
  * @package WordPress
  */
 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
```
Comment (mukeshpanchal27) [inline src/wp-includes/blocks/index.php]:
```suggestion
// Don't load directly.
if ( ! defined( 'ABSPATH' ) ) {
	die( '-1' );
}
```

Core use above code for checking. For consistency can we use that?

Comment (itzmekhokan) [reply src/wp-includes/blocks/index.php]: @mukeshpanchal27 suggestion updated.

**Review by mukeshpanchal27 — APPROVED** [review]

**Comment by mukeshpanchal27** [comment]
As mention in ticket description there is already a PR #6635

**Comment by mukeshpanchal27** [comment]
Wrong one 😓

**Comment by peterwilsoncc** [comment]
Merged [r59117](https://core.trac.wordpress.org/changeset/59117) / https://github.com/WordPress/wordpress-develop/commit/7e30d693ae42f805da1bd9cac6231db48129223c

**Pull request closed without merging**